azure-sink-connector/src/integration-test/java/io/aiven/kafka/connect/azure/sink/AvroIntegrationTest.java

```diff
@@ -303,7 +299,7 @@ void jsonlOutput() throws ExecutionException, InterruptedException {
         final List<String> expectedBlobs = Arrays.asList(getBlobName(0, 0, compression), getBlobName(1, 0, compression),
                 getBlobName(2, 0, compression), getBlobName(3, 0, compression));
         awaitAllBlobsWritten(expectedBlobs.size());
-        assertIterableEquals(expectedBlobs, testBlobAccessor.getBlobNames(azurePrefix));
+        assertThat(testBlobAccessor.getBlobNames(azurePrefix)).containsExactlyElementsOf(expectedBlobs);
```

Comment:
Is there any chance the blobnames can be in a different order? Should we always use  ".containsExactlyInAnyOrderElementsOf" 

Same question to the use of isEqualTo on 271 is there any chance ordering could lead to a flakey test and we should switch?

Reply:
To answer my own question it looks like the assert that is being replaced requires the same ordering
https://github.com/junit-team/junit5/blob/a9b0a129a1ed48c635bdc58d66f3bbfb2165ddf2/junit-jupiter-api/src/main/java/org/junit/jupiter/api/AssertIterableEquals.java#L54